Sources/IdentityState.swift

```diff
@@ -104,8 +104,8 @@ class IdentityState {
     ///
     /// - Parameters
     ///   - event: event containing customer identifiers to add or update with the current customer identifiers
-    ///   - createXDMSharedState: function which creates new XDM shared state
-    func updateCustomerIdentifiers(event: Event, createXDMSharedState: ([String: Any], Event) -> Void) {
+    ///   - resolveXDMSharedState: function which resolves pending XDM shared state
+    func updateCustomerIdentifiers(event: Event, resolveXDMSharedState: ([String: Any]) -> Void) {
         guard let identifiersData = event.data else {
             Log.debug(label: IdentityConstants.FRIENDLY_NAME, "IdentityState - Failed to update identifiers as no identifiers were found in the event data.")
             return
```

Comment:
The pending shared state needs to be resolved before returning from this function. At any point where this function can return, resolve the shared state by passing in the current Identity properties.

Reply:
Add unit tests for these cases as well. Verify that the pending shared state is resolved after the update or remove customer identifiers methods are called, even when passed no or invalid identifiers to update/remove.